kyaml/yaml/types.go

```diff
@@ -517,6 +518,36 @@ func String(node *yaml.Node, opts ...string) (string, error) {
 	return val, errors.Wrap(err)
 }
 
+// RearrangeComments addresses a bug in yaml V3 upstream, it parses the yaml node,
```

Comment:
Consider putting this in its own file. -- `serialization.go`

Reply:
Done